packages/visual-editor/src/components/pageSections/Banner.tsx

```diff
@@ -86,19 +86,21 @@ const BannerComponent = ({ data, styles }: BannerSectionProps) => {
   }[styles.textAlignment];
 
   return (
-    <PageSection
-      background={styles.backgroundColor}
-      verticalPadding="sm"
-      className={`flex ${justifyClass} items-center`}
-    >
-      <EntityField
-        displayName={pt("fields.bannerText", "Banner Text")}
-        fieldId={data.text.field}
-        constantValueEnabled={data.text.constantValueEnabled}
+    resolvedText && (
```

Comment:
Might be simpler to do

```
if (!resolvedText) { return <></>; }
return <PageSection...
```

Then we can do the same in a bunch of other places eventually. Though maybe that changes if we still need to return the full thing wrapped in some Invisible component